base_station/gui/src/components/SATask.vue

```diff
@@ -65,7 +65,6 @@ import Cameras from './Cameras.vue'
 import CommIndicator from './CommIndicator.vue'
 import DriveControls from './DriveControls.vue'
 import Raman from './Raman.vue'
-import WaypointEditor from './WaypointEditor.vue'
```

Comment:
@jnnanni I believe you're referring to these changes. An argument could be made for making a separate PR to remove these lines since it's not related to auton but this PR includes a lot of renaming/clean up around the map/waypoint editor so I think it's fine.